### ISSUE TITLE
[Suggestion] % completion bar

### ISSUE THREAD

**Issue by ManDude** [larsiusprime/tdrpg-bugs#899]
This was already discussed on the forums at one point, but I thought that I should put it here just in case.

The game should have a bar with XX% written on it to indicate game completion somewhere in the save slot or the achievements menu. Game completion goes up when you get gold stars, blue stars, earn unique equipment, earn books and upgrade unique equipment.

**Comment by dqhol** [comment]
Agreed completely. If a person gets to the end of Normal Play and sees their file only has about 40% completion, they'll be VERY intrigued about NG+, I wager.
